include/Realm.h

```diff
@@ -448,6 +450,8 @@ class Realm {
   Actuator *actuator_;
   std::shared_ptr<ActuatorMetaFAST> actuatorMeta_;
   std::shared_ptr<ActuatorBulkFAST> actuatorBulk_;
+  std::shared_ptr<ActuatorMetaSimple> actuatorMetaSimple_;
+  std::shared_ptr<ActuatorBulkSimple> actuatorBulkSimple_;
```

Comment:
Why do we need additional entries? Can we not have a base class that can handle both these types?

Reply:
Yes, I agree, we should be able to do to this cleaner.  See phil's comment [here](https://github.com/psakievich/nalu-wind/pull/13#discussion_r407708831) -- some sort of shared pointer for the structures and some dynamic casts.  I'm thinking we should do this change in another PR because it will affect all actuator NGP codes.